__tests__/app.test.js

```diff
@@ -45,4 +45,55 @@ describe("NC-Games app", () => {
         });
     });
   });
+  describe("GET /api/reviews/:review_id", () => {
+    test("Returns 200, responds with a single review.", () => {
```

Comment:
You can combine this test with the next test block! You can check if body.review is NOT an array.